docs/en/e-service-pdnd-catalogue.rst

```diff
@@ -104,3 +104,29 @@ Get Attribute Claims
       - Authentic Source 
     * - **Consumer**
       - PID/(Q)EAA Provider
+
+.. _wp-catalogue:
+
+Wallet Provider Catalogue
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+User's death MUST lead to the revocation of the Wallet Instances of the User and the deletion of the User account at the Wallet Provider. For this reason, the Wallet Provider MUST provide the following e-service through PDND.
```

Comment:
```suggestion
User's death leads to the revocation of the Wallet Instances of the User and the deletion of the User account at the Wallet Provider. For this reason, the Wallet Provider MUST provide the following e-service through PDND.
```

we cannot put normative language to the user death!

we cannot pretend anymore from a death user or from the death intended as an supernatural entity!